src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -1,16 +1,105 @@
 // @flow
 import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+import noop from 'lodash/noop';
+
+import LoadingIndicator from '../../../../components/loading-indicator';
+import PlainButton from '../../../../components/plain-button';
+import ActivityThreadReplies from './ActivityThreadReplies';
+
+import type { GetAvatarUrlCallback, GetProfileUrlCallback } from '../../../common/flowTypes';
+import type { Translations } from '../../flowTypes';
+import type { SelectorItems, User } from '../../../../common/types/core';
+import type { Comment as CommentType } from '../../../../common/types/feed';
+
+import messages from './messages';
+import './ActivityThread.scss';
 
 type Props = {
     children: React.Node,
+    currentUser?: User,
+    getAvatarUrl: GetAvatarUrlCallback,
+    getMentionWithQuery?: Function,
+    getUserProfileUrl?: GetProfileUrlCallback,
     hasReplies: boolean,
+    mentionSelectorContacts?: SelectorItems<>,
+    onGetReplies?: () => void,
+    onReplyDelete?: Function,
+    onReplyEdit?: Function,
+    replies?: Array<CommentType>,
+    repliesLoading?: boolean,
+    totalReplyCount?: number,
+    translations?: Translations,
 };
 
-const ActivityThread = ({ children, hasReplies }: Props) => {
-    if (hasReplies) {
-        return <div className="bcs-ActivityThread">{children}</div>;
+const ActivityThread = ({
+    children,
+    currentUser,
+    getAvatarUrl,
+    getMentionWithQuery,
+    getUserProfileUrl,
+    hasReplies,
+    mentionSelectorContacts,
+    onGetReplies = noop,
+    onReplyDelete = noop,
+    onReplyEdit = noop,
+    replies = [],
+    repliesLoading,
```

Comment:
Minor, but typically we prefix boolean flags with `is` or `has`. So, `isLoadingReplies` or something similar.